packages/esm-patient-programs-app/src/programs/programs-detailed-summary.component.tsx

```diff
@@ -86,10 +86,16 @@ const ProgramsDetailedSummary: React.FC<ProgramsDetailedSummaryProps> = ({ patie
   }, [enrollments, t]);
 
   const launchProgramsForm = useCallback(() => launchPatientWorkspace('programs-form-workspace'), []);
+  const noCompletedPrograms = useMemo(() => {
+    return enrollments?.every((program) => !program.dateCompleted);
+  }, [enrollments]);
```

Comment:
How about something like this instead for determining whether a patient is ineligible for new enrollments or re-enrollments to completed programs:

```tsx
  const isEnrolledInAllPrograms = useMemo(() => {
    if (!availablePrograms?.length || !enrollments?.length) {
      return false;
    }

    const activeEnrollments = enrollments.filter((enrollment) => !enrollment.dateCompleted);
    return activeEnrollments.length === availablePrograms.length;
  }, [availablePrograms, enrollments]);
```